cmp/tab/TabModel.ts

```diff
@@ -91,30 +94,42 @@ export class TabModel extends HoistModel {
     showRemoveAction: boolean;
     content: Content;
 
+    containerModel: TabContainerModel;
+    @managed refreshContextModel: RefreshContextModel;
+
     private _renderMode: RenderMode;
     private _refreshMode: RefreshMode;
+    private _childTabsRef: RefObject<TabContainerModel> = createObservableRef();
```

Comment:
I would rename this `_childTabContainerModelRef` since `_childTabsRef` sounds like it's a ref to a collection of `TabModels` rather than a `TabContainerModel`

Reply:
going with `childContainerModel`, for symmetry with existing `containerModel`